File: Entidades/ErrorIntegridadEN.vb
Public Class ErrorIntegridadEN
    Private _coden As Integer
    Public Property CodEn() As Integer
        Get
            Return _coden
        End Get
        Set(ByVal value As Integer)
            _coden = value
        End Set
    End Property

    Private _tabla As String
    Public Property Tabla() As String
        Get
            Return _tabla
        End Get
        Set(ByVal value As String)
            _tabla = value
        End Set
    End Property

    Private _tipo As String
    Public Property Tipo() As String
        Get
            Return _tipo
        End Get
        Set(ByVal value As String)
            _tipo = value
        End Set
    End Property

    Private _estadomensaje As Boolean
    Public Property EstadoMensaje() As Boolean
        Get
            Return _estadomensaje
        End Get
        Set(ByVal value As Boolean)
            _estadomensaje = value
        End Set
    End Property



End Class

File: Entidades/ErrorIntegridadEN.cs

namespace Entidades
{
    public class ErrorIntegridadEN
    {
        private int _coden;

        public int CodEn
        {
            get
            {
                return _coden;
            }

            set
            {
                _coden = value;
            }
        }

        private string _tabla;

        public string Tabla
        {
            get
            {
                return _tabla;
            }

            set
            {
                _tabla = value;
            }
        }

        private string _tipo;

        public string Tipo
        {
            get
            {
                return _tipo;
            }

            set
            {
                _tipo = value;
            }
        }

        private bool _estadomensaje;

        public bool EstadoMensaje
        {
            get
            {
                return _estadomensaje;
            }

            set
            {
                _estadomensaje = value;
            }
        }
    }
}